release/models/system/openconfig-system.yang

```diff
@@ -1255,6 +1262,7 @@ module openconfig-system {
       uses oc-messages:messages-top;
       uses oc-license:license-top;
       uses system-macaddr-top;
+      uses oc-sys-red:system-redundancy-top;
```

Comment:
I would suggest going fwd that specific features like this be treated as augments rather than direct import/uses as this gives implementations flexibility to support portions of model sets vs. having to import everything then deviate it out.  This should go back into design guidelines as such as well.

To a consumer, it is highly confusing if an implementation advertises support for all models and then has to negate them by way of deviations (The network-instance problem statement)

Reply:
Updated model to augment instead of direct import/uses.